### DISCUSSION TITLE
How to set the AG-Grid row hover color

### DISCUSSION THREAD

**Discussion by snowbollaanm** [zauberzeug/nicegui#2206] [Q&A]
### Description

Hey I've found a strange issue with AG Grid when trying to change the row hover color. 

I've changed the hover color for my rows and I believe its running the CSS correctly because for about the first half second it applies the correct color and switches to the default background color. 

Here's a minimal reproducible example:

```py
from nicegui import ui 

ui.add_body_html('''
<style>
    .ag-theme-material .ag-row-hover  { background-color: rgb(255, 0, 49); }     
</style>         
''')

grid = ui.aggrid({
    'defaultColDef': {'flex': 1},
    'columnDefs': [
        {'headerName': 'Name', 'field': 'name'},
        {'headerName': 'Age', 'field': 'age'},
        {'headerName': 'Parent', 'field': 'parent', 'hide': True},
    ],
    'rowData': [
        {'name': 'Alice', 'age': 18, 'parent': 'David'},
        {'name': 'Bob', 'age': 21, 'parent': 'Eve'},
        {'name': 'Carol', 'age': 42, 'parent': 'Frank'},
    ],
    'rowSelection': 'multiple',
}, theme='material')

ui.run()
```

From what I can tell online this is the standard way to change the color. Strangely changing the header hover color works correctly:
```
    ui.add_body_html('''
    <style>
    .ag-theme-material {
        --ag-header-cell-hover-background-color: rgb(131, 131, 131);
    }     
    </style>        
    ''')
```

**Comment by falkoschindler** [comment]
Hi @snowbollaanm, I'm not sure what is causing this color switch. But [here](https://www.ag-grid.com/javascript-data-grid/row-styles/#styling-the-first-and-last-rows) I noticed that they don't style the row background color, but the cell. And indeed, this works:

```py
ui.add_body_html('<style>.ag-row-hover .ag-cell  { background-color: red }</style>')
ui.aggrid({
    'columnDefs': [{'headerName': 'Name', 'field': 'name'}],
    'rowData': [{'name': 'Alice'}, {'name': 'Bob'}, {'name': 'Carol'}],
})
```
